error: ‘isinf’ was not declared in this scope

Does not build on Arch Linux ( g++ 5.3 )

Problem is with file `glslang/MachineIndependent/intermOut.cpp`
and it looks like it's because you use `isinf` instead of `std::isinf` ( because `isinf` is in `std` namespace ) - http://en.cppreference.com/w/cpp/numeric/math/isinf


Full error:

```
[ 34%] Building CXX object glslang/CMakeFiles/glslang.dir/MachineIndependent/intermOut.cpp.o
***/glslang/glslang/MachineIndependent/intermOut.cpp: In function ‘bool {anonymous}::is_positive_infinity(double)’:
***/glslang/glslang/MachineIndependent/intermOut.cpp:54:17: error: ‘isinf’ was not declared in this scope
   return isinf(x) && (x >= 0);
                 ^
***/glslang/glslang/MachineIndependent/intermOut.cpp:54:17: note: suggested alternative:
In file included from /usr/include/c++/5.3.0/random:38:0,
                 from /usr/include/c++/5.3.0/bits/stl_algo.h:66,
                 from /usr/include/c++/5.3.0/algorithm:62,
                 from ***/glslang/glslang/MachineIndependent/../Include/../Include/Common.h:78,
                 from***/glslang/glslang/MachineIndependent/../Include/intermediate.h:49,
                 from ***/glslang/glslang/MachineIndependent/localintermediate.h:38,
                 from***/glslang/glslang/MachineIndependent/intermOut.cpp:37:
/usr/include/c++/5.3.0/cmath:621:5: note:   ‘std::isinf’
     isinf(_Tp __x)
     ^
```

--- a/glslang/MachineIndependent/intermOut.cpp
+++ b/glslang/MachineIndependent/intermOut.cpp
@@ -48,7 +48,7 @@ namespace {
 bool is_positive_infinity(double x) {
 #ifdef _MSC_VER
   return _fpclass(x) == _FPCLASS_PINF;
-#elif defined __ANDROID__
+#elif defined __ANDROID__ || defined __linux__
   return std::isinf(x) && (x >= 0);
 #else
   return isinf(x) && (x >= 0);

